org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.java

```diff
@@ -290,6 +290,8 @@ public void checkWidgets(SelectionEvent e) {
 		Object source = null;
 		if (e != null) {
```

Comment:
Reverse logic and keep source in one assignment

Reply:
Fixed.